ClientConfiguration/SupportedVersion.json

```diff
@@ -0,0 +1,23 @@
+{
+  "FormatVersion": "1.0",
+  "Versions": [
+    {
+      "DisplayName": ".NET 6 (Microsoft LTS)",
+      "VersionKey": "net6.0",
+      "RequiredVisualStudioVersion": "17.0.0",
+      "RecommendOrder": "1"
+    },
+    {
+      "DisplayName": ".NET Core 3.1 (Microsoft LTS)",
```

Comment:
Do we have any mechanism to remind us to update this to "Microsoft out of support" once 3.1 (and any future runtime) goes out of support?

Reply:
No automated mechanism so far, this will be manual process. I thought about adding some information, which client can use to logically tell if it's out of support or not. Just updating the display name anytime, we need, and the client UI will pick this up.

Reply:
Another reason is that I want to make this display name more flexible, in case of we want to name them differently from a few logical selections.

Reply:
Maybe you can just create a reminder, maybe on Slack? Because 3.1 goes out of support soon. Maybe we check this every other month or something until we have a better solution.

Reply:
Got it. A reminder is good approach. Will do that.

Reply:
I will create a Slack reminder afterwards.